Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Panels/ToggleFeaturesPanel.prefab

```diff
@@ -398,6 +387,11 @@ PrefabInstance:
   m_Modification:
     m_TransformParent: {fileID: 8503270331930508642}
     m_Modifications:
+    - target: {fileID: 538639403742340272, guid: 9215a7c858170d74fb2257375d5feaf1,
```

Comment:
Make sure you've pulled the latest changes to this file. Will made some edits recently you might be squashing over.

Reply:
Whenever i get conflicts on asset files i usually avoid trying to merge and just checkout from mrtk_development, then redo my changes in the editor. There may have been some sneaky auto-merge here, i'll check again.

Reply:
This should be alright. The only actual change i did was removing the "SaveRecordedInput" target from the OnRecordingStopped event of the ToggleFeaturesPanel, and add it as a second action to the ToggleInputRecording button. The reason is that i want to avoid saving recorded input twice, via the ToggleFeaturesPanel and then the recording button in the editor tools.

The rest of the changes seem to be some kind of automated Unity fluff that i have no control over. In any case, it's based on latest mrtk_development, so should not affect any of Yoon's changes.